hopo/studiengebuehren.tex

```diff
@@ -78,25 +157,76 @@ \subsection*{Verwendung der QuaSiMi}
 \iffalse
```

Comment:
Wie das ganze mit iffalse in dem Dokument funktioniert verstehe ich auch noch nicht so ganz, bzw. was man damit bezwecken wollte.